src/components/List/RowContent.js

```diff
@@ -347,7 +398,30 @@ const ListRowContent = ({
               {isOpen && onExpandShow === "title" ? title : subTitle}
             </MultilineText>
           </ContentColumn>
-        </Row>
+
+          <WithOptions variant={variant}>
```

Comment:
Can you please move this `WithOptions` and `children` into a separate file? The markup in this return statement is becoming too large.

Reply:
Done.